### ISSUE TITLE
Warning: React.render is deprecated

### ISSUE THREAD

**Issue by pwldp** [meteor/react-packages#141]
Hi,
After run new app I got warning:
Warning: React.render is deprecated. Please use ReactDOM.render from require('react-dom') instead.

**Comment by lawwantsin** [comment]
I have the same issue on the Meteor docs example of all things.  Way to break stuff MDG.

**Comment by Un3qual** [comment]
+1

**Comment by hlandao** [comment]
use ReactDOM.render instead of React.render

**Comment by yyx990803** [comment]
Since React 0.14, you should use `ReactDOM.render()` instead of `React.render()`. The tutorial is a bit outdated, which we will fix soon.

**Comment by iboxgithub** [comment]
Am I the only one to still have this error? Is this issue really closed?

**Comment by TimeBandit** [comment]
Nope its not closed, i saw it today :)

**Comment by stubailo** [comment]
What is the stack trace? Some of your code, or a package you are using, must be calling `React.render` somewhere.

**Comment by TimeBandit** [comment]
It is the react todo tutorial which is in error

**Comment by stubailo** [comment]
> The tutorial is a bit outdated, which we will fix soon.

**Comment by firrae** [comment]
I am using the react-template-helper package and getting the issue as well for all aspects related to ReactDOM (render, componentDidUnmount, etc.)

**Comment by firrae** [comment]
So apparently 3 of us at work all installed both the React and React-Template-Helper packages today and I got older version of both while theirs downloaded the version previous to the most recent version on Atmosphere. There seems to be something weird going on in this regard but after forcing those packages to update the problem has gone away.

Specifically on Jan. 6 I got React 0.14.1 when installing it, and react-template-helper 0.1.1X, the other 2 in my office got React 0.14.1_1 and react-template-helper 0.2.3.
